.babelrc

```diff
@@ -1,17 +1,17 @@
 // .babelrc
 {
   "presets": [
-    ["es2015", {"modules": false}],
-    "react"
-  ],
-  "plugins": [
-    "transform-object-assign",
-    "transform-class-properties",
-    "react-hot-loader/babel"
-  ],
-  "env": {
-    "test": {
-      "plugins": ["transform-es2015-modules-commonjs"]
-    }
-  }
+    "@babel/preset-react",
+    [
+      "@babel/preset-env",
+      {
+          "useBuiltIns": "entry",
+          "corejs": "3.19",
+          "targets": {
+              "chrome": 75,
+              "ios": 12
+          }
+      }
+    ]
+  ]
```

Comment:
Is this config change needed due to upgrading webpack?

Reply:
Some of the existing babel dependencies were incompatible - since I was moving from `babel-core` et al to `@babel/core` et all I needed to overhaul this file - mostly this is borrowed from Composer's `.babelrc`